docs/pages/database-access/guides/postgres-redshift.mdx

```diff
@@ -137,6 +137,10 @@ $ tsh db ls
 
 </ScopedBlock>
 
+<Admonition type="note" title="Note">
```

Comment:
This part of the page has to do with listing the available databases before you connect to one, so I'm not sure it's the best place to add instructions for overriding the name of a specific database. 

Also, is this an AWS label or a Teleport label? (i.e., how would you apply it?)

Reply:
AWS label, so you can do it with AWS tools.

I'm open to suggestions for better placement?